src/utils/fs.spec.ts

```diff
@@ -17,14 +18,15 @@ describe('utils/fs module', () => {
 
     test('should throw if failing', async () => {
       try {
+        // todo: we are physically creating a folder called "url*path", we should mock the creation of the folder since this is a unit test
         await createFolder('url*path')
       } catch (e: any) {
         expect(e.message).to.be.contain('ENOENT: no such file or directory')
       }
     })
-
+    // todo: it looks to me that the shape of createFolder has been changed , because right now it only includes one parameter, so failSilently parámeteris missing
     test('should return false if failing and failSilently is set to true', async () => {
-      const result = await createFolder('url*path', true)
```

Comment:
I've decided to go against initial failSilently option which was there to handle plugin failures. In my opinion, if you use this plugin then use it in working mode, or don't use it at all.

And the obnoxious ALLOW_VISUAL_REGRESSION_TO_FAIL (or smth like that) is now simply failSilently.

Reply:
👍